CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
   [#106](https://github.com/oddbird/sassdoc-theme-herman/issues/106)
 - BREAKING: Group navigation is ordered according to config `groups` order –
   [#150](https://github.com/oddbird/sassdoc-theme-herman/issues/150)
+- BUGFIX: Do not error if `sass` config option is `undefined`.
+- BUGFIX: Do not break layout on fonts with single-item stack –
+  [#172](https://github.com/oddbird/sassdoc-theme-herman/issues/172)
+- BREAKING: `@preview font-specimen` is replaced by new `@font` annotation.
+- BREAKING: Removed `customHead` option; replaced by multiline `@font`
+  annotation.
```

Comment:
A few subtle things that should be documented related to the `@font` annotation:

1) The font family name will default to the name of the font-map unless a `name` key is found.
2) If the font-map contains an `svgid` key, that value will be used for the `@font-face` `svg-file.svg#svgid` suffix. If none is found, it will default to the font family name.
3) If webfonts require additional HTML (e.g. external Google stylesheet or Typekit JS), they must be initialized with the `@font` annotation for them to work properly in `@example` annotations. The custom HTML defined in the `@font` annotation is automatically inserted into the `@example` annotation `<head>`.